pkgs/by-name/pa/packet/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cargo,
+  meson,
+  ninja,
+  pkg-config,
+  protobuf,
+  rustPlatform,
+  rustc,
+  wrapGAppsHook4,
+  cairo,
+  dbus,
+  gdk-pixbuf,
+  glib,
+  gtk4,
+  libadwaita,
+  pango,
+  blueprint-compiler,
+  desktop-file-utils,
+  appstream,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "packet";
+  version = "0.3.4";
+
+  src = fetchFromGitHub {
+    owner = "nozwock";
+    repo = "packet";
+    tag = finalAttrs.version;
+    hash = "sha256-s3R/RDfQAQR6Jdehco5TD+2GpG4y9sEl0moWMxv3PZE=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit (finalAttrs) src;
```

```suggestion
    inherit (finalAttrs) pname version src;
```

This will force hash to change even if someone forgets

---

Oh I just hadn't seen